js-jasmine/spec/gilded_rose_spec.js

```diff
@@ -1,10 +1,94 @@
-var {Shop, Item} = require('../src/gilded_rose.js');
+const BackstagePass = require('../src/backstage_passes')
+const AgedBrie = require('../src/aged_brie')
+const Sulfuras = require('../src/sulfuras')
+const NormalItem = require('../src/normal_item')
+const ConjuredItem = require('../src/conjured_item')
+const Item = require('../src/item')
+
 describe("Gilded Rose", function() {
```

Comment:
possible suggestion: you could have a different spec file for each class so you can test each thing separately

Reply:
Yeah I think I'll do this!